Makefile.PL

```diff
@@ -74,3 +75,37 @@ WriteMakefile(
   dist => { PREOP=>'$(PERL) -MPDL::Core::Dev -e pdlpp_mkgen $(DISTVNAME)' }, # GENERATED subdir in dist tarball
   clean => { FILES => '*~' },
 );
+
+
+sub find_libs {
+  return if $^O !~ /linux/i;
+
+  my @dirs = split /\s+/, `ldconfig -Nv 2>/dev/null | grep : | cut -f1 -d:`;
+
+  # in performance order based on libraries I've used
+  # for xnec2c in Ubuntu, Debian, SuSE, CentOS, etc.
+  # See comments here for detail:
+  #    https://github.com/KJ7LNW/xnec2c/blob/master/src/mathlib.c#L29
+  my @libs = qw/
+      libopenblaso.so.0 libopenblaso.so libopenblas_openmp.so.0
+      libtatlas.so.3
+      liblapack_atlas.so.3
+      libopenblasp.so.0 libopenblasp.so libopenblas_pthreads.so.0
+      libopenblas.so.0 libopenblas.so libopenblas_serial.so.0
+      libsatlas.so.3
+      liblapacke.so.3
+      libmkl_rt.so/,
+    ;
```

Comment:
If using `Devel::CheckLib`, this does not need all the suffixed versions of the `.so`, just the name without the `lib` prefix and the `/.so.*$/` suffix.

Reply:
Updated with Devel::CheckLib